piqe_ocp_lib/tests/resources/test_ocp_nodes.py

```diff
@@ -340,6 +340,30 @@ def test_mark_node_schedulable(self, setup_params):
             api_response = node_api_obj.mark_node_schedulable(node_name=worker_node_name.metadata.name)
             assert api_response.kind == "Node"
 
+    def test_make_node_uncordon(self, setup_params):
```

Comment:
The test should return the node to its original state. I think we can either add a method that does the opposite and provide a test for it or simply clean up after the test. I am leaning towards the first option.

Reply:
okay, so we should write a method to cordon the node and then uncordon it again to convert it into its original state?

Reply:
Correct. I think that's the best resolution.